packages/core-typings/src/IRoom.ts

```diff
@@ -124,6 +124,8 @@ export const isDiscussion = (room: Partial<IRoom>): room is IRoom => !!room.prid
 export const isPrivateDiscussion = (room: Partial<IRoom>): room is IRoom => isDiscussion(room) && room.t === 'p';
 export const isPublicDiscussion = (room: Partial<IRoom>): room is IRoom => isDiscussion(room) && room.t === 'c';
 
+export const isPublicRoom = (room: Partial<IRoom>): room is IRoom => room.t === 'c';
```

Comment:
I know that you are just following the pattern from the functions above, but none of them are really good, receiving a partial and assuring this is a full complete type just because we tested one prop is not a good thing